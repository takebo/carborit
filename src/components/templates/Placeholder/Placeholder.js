import React from 'react';
import PropTypes from 'prop-types';
import './Placeholder.scss';

const Placeholder = props => {
  const bgImg = {
    backgroundImage: 'url(' + props.placeholderImage + ')',
  };
  return (
    <div className="page-placeholder" style={bgImg}>
      <p>{props.placeholderText}</p>
    </div>
  );
};

export default Placeholder;

Placeholder.propTypes = {
  placeholderImage: PropTypes.string,
  placeholderText: PropTypes.string,
};
