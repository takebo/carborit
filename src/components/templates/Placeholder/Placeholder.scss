.page-placeholder {
  background-size: auto 90%;
  background-position: center bottom;
  background-repeat: no-repeat;
  width: 100%;
  height: 460px;
  p {
    font-size: 24px;
    color: #c4c4c4;
    font-weight: 300;
    text-align: center;
  }
}
