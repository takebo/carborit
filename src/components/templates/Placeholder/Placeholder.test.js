import React from 'react';
import { shallow } from 'enzyme';

import Placeholder from './Placeholder';

describe('PLaceholder', () => {
  it('should render', () => {
    const props = {
      placeholderText: '',
      placeholderImage: '',
    };
    const wrapper = shallow(<Placeholder {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
