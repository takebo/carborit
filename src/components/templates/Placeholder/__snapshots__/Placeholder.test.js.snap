// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLaceholder should render 1`] = `
<div
  className="page-placeholder"
  style=""
>
  <p />
</div>
`;
